### PR TITLE
Update React Native snapshot

### PR DESCRIPTION
**Pull request by samuelfullerthomas**
See: https://github.com/paypal/downshift/pull/451

Also re-add `onChange`. Question - should there be something in the README about the RN only tests? Is there? Did I miss it?

### CODE REVIEW

**Comment by samuelfullerthomas** [comment]
@kentcdodds

File: other/react-native/__tests__/__snapshots__/render-tests.js.snap [hunk 1]
```diff
@@ -19,7 +19,6 @@ exports[`renders with React Native components 1`] = `
     onBlur={[Function]}
     onChange={[Function]}
     onChangeText={[Function]}
-    onKeyDown={[Function]}
```
Comment (kentcdodds) [inline other/react-native/__tests__/__snapshots__/render-tests.js.snap]: Do we not need this?

Comment (samuelfullerthomas) [reply other/react-native/__tests__/__snapshots__/render-tests.js.snap]: Nope!

See https://github.com/paypal/downshift/pull/451#discussion_r215278302

Comment (samuelfullerthomas) [reply other/react-native/__tests__/__snapshots__/render-tests.js.snap]: This lists the valid props for RN Text Input - no onKeyDown: https://facebook.github.io/react-native/docs/0.56/view#props

**Pull request merged**